src/Api/V1/Engine/Api.php

```diff
@@ -509,10 +509,29 @@ private static function outputXML($statusCode, array $data = null)
         array_walk($data, array(__CLASS__, 'arrayToXML'), $root);
 
         // set correct headers
-        \SpoonHTTP::setHeadersByCode($statusCode);
-        \SpoonHTTP::setHeaders('content-type: text/xml;charset=' . $charset);
+        header('HTTP/1.1 ' . self::getDecentHeaderMessage($statusCode));
+        header('content-type: text/xml;charset=' . $charset);
 
         // output XML
         self::$content = $XML->saveXML();
     }
+
+    private static function getDecentHeaderMessage($statusCode)
+    {
+        $possibleCodes = array(
```

Comment:
It seems more logical to me to use this array: https://github.com/symfony/symfony/blob/a44ceb3156e549ffc17baaab7a206be39b636d4f/src/Symfony/Component/HttpFoundation/Response.php#L125. It's more complete and we already have this code loaded trough composer. No need to reinvent the weel :)